lib/phoenix/router.ex

```diff
@@ -121,10 +121,10 @@ defmodule Phoenix.Router do
       "/pages/hello?some=query"
 
       MyAppWeb.Router.Helpers.page_url(conn_or_endpoint, :show, "hello")
-      "http://example.com/pages/hello"
+      "https://example.com/pages/hello"
```

Comment:
I do not think we should change this URL– the current value matches the default Endpoint configuration from the Phoenix installer.
```suggestion
      "http://example.com/pages/hello"
```